ci/jenkins/test-vmc.sh

```diff
@@ -483,14 +483,14 @@ function run_integration {
       # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
       ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "PATH=$PATH:/usr/local/go/bin && umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && cd multicluster && NO_LOCAL=true make test-integration"
       if [[ "$COVERAGE" == true ]]; then
-        run_codecov "mc-integration-tests" "antrea/multicluster/.coverage/coverage-integration.txt" "" true ${VM_IP}
+        run_codecov "mc-integration-tests" "coverage-integration.txt" "multicluster/.coverage" true ${VM_IP}
```

Comment:
does it work, `dir` is not even used when `remote` is true.

Reply:
oh, yes, I think we can leave this directory empty in `remote` case, it's not used, I will update it.

Reply:
Removed it.